src/project/project-hapi-plugin.ts

```diff
@@ -28,8 +28,13 @@ export default class ProjectHapiPlugin {
 
   public register: HapiRegister = (server, _options, next) => {
     server.route({
-      method: 'GET',
+      method: 'POST',
```

Comment:
Yes, it appears that this was incorrect before. We don't currently use the system hook for anything (we only use project hooks), so this was not noticed.